change/@azure-msal-browser-03d446bd-1961-4473-b890-89912abebee8.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Adding the logger to the CacheManager makes this minor and you'll need to update the 1P lib as well
```suggestion
  "type": "minor",
```